Code/Framework/AzFramework/AzFramework/Windowing/WindowBus.h

```diff
@@ -55,6 +55,10 @@ namespace AzFramework
         //! Set the client area size. This is the size that can be rendered to.
         virtual void ResizeClientArea(WindowSize clientAreaSize) = 0;
 
+        //! Does this platform support window resizing.
+        //! Generally desktop platforms support resizing, mobile platforms don't.
+        virtual bool SupportsClientAreaResize() const = 0;
```

Comment:
style: do we want to assign `false` here rather than implicit cast int?

Reply:
@smurly This isn't an assignment.
It is a grammar construct of C++ to signify that the SupportsClientAreaResize function is a "pure virtual function".

You can read here for more information on pure virtual functions and abstract classes: https://en.cppreference.com/w/cpp/language/abstract_class